docs/development/testguide.rst

```diff
@@ -117,6 +100,27 @@ This can be used in conjunction with the ``-P`` option provided by the
 `pytest-filter-subpackage <https://github.com/astropy/pytest-filter-subpackage>`_
 plugin to run just part of the test suite.
 
+Using tox to set up a development environment
+---------------------------------------------
+
+As noted, running ``tox`` can be quite slow, yet it has the advantage over the
+other methods that it puts together a consistent set of dependencies.  One can
```

Comment:
In general use "you" not "one".
https://docs.astropy.org/en/stable/development/style-guide.html#a-note-about-style-and-tone